### ISSUE TITLE
AutoTable with Nested HTML

### ISSUE THREAD

**Issue by lorengphd** [simonbengtsson/jsPDF-AutoTable#416]
I have updated to the latest version of AutoTables v2.3.5. I am using the tool to generate a PDF out of an HTML table. However, some of my table cells have embedded `<ol>` tags in them. In the PDF they are coming out as raw text. I can see the HTML tags.  Additionally, I have some cells that have `<p>` tags with the same issue.

Any recommendations for rendering these tags rather than displaying raw text? I have seen your assistance with rendering images. I attempted to make these recommendations work for HTML with no luck.

Thanks!

**Comment by lorengphd** [comment]
I have been digging into this further. I've noticed that there's a function `autoTableHtmlToJson`. I am unable to use because I am receiving my data via json (not an actual HTML table). 

However, I've copied the functionality of this function, namely converting the values to each cell into an HTMLElement. Now when I pass that data to `autoTable` I get a pdf. But the values in each cell aren't formatted. (I've inspected and noticed this is because autoTable ultimately calls `innerText` on the HtmlElements).

For example, I'd like send a value to a cell that looks like "this is a <strong>bold</strong> word".  Or "<ul><ol>Bulleted line</ol></ul>"

In older versions of jspdf Autotables it worked when I sent those values _as a string_. Now when I send that as a string is doesn't apply the styling, but rather displays the raw HTML.
